ios/engine/KMEI/KeymanEngineTests/MigrationTests.swift

```diff
@@ -15,6 +15,98 @@ class MigrationTests: XCTestCase {
     TestUtils.standardTearDown()
   }
 
+  // Uncomment this and set up whatever resources are needed for your test bundle in order to build one!
+  func testForMigrationBundleConstruction() {
```

Comment:
Forgot to comment this back out.  Though... as written, it actually doesn't break anything.  (The constructed bundle becomes an attachment to the test report; it doesn't have permanent collateral effects.)

But yeah, still want to comment it back out.